### PR TITLE
Use single quotes in JS files

### PR DESCRIPTION
**Pull request by ashmaroli**
Replace all use of double-quotes with single quotes in all functional JS files following convention.

The test files have not been altered as they may contain strings that use backslashes to escape special characters or may contain an `apostrophe`

### CODE REVIEW

**Comment by ashmaroli** [comment]
Aww snap! Apparently, ESLint expects `double-quotes` in `JSX`

File: src/components/Breadcrumbs.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default class Breadcrumbs extends Component {
     });
 
     return (
-      <ul className="breadcrumbs">
+      <ul className='breadcrumbs'>
```
Comment (mertkahyaoglu) [inline src/components/Breadcrumbs.js]: Let's not use single quote for html attributes, only javascript.

File: package.json [hunk 2]
```diff
@@ -184,7 +184,7 @@
       "no-underscore-dangle": 0,
       "no-alert": 0,
       "no-lone-blocks": 0,
-      "jsx-quotes": 1,
+      "jsx-quotes": ["error", "prefer-single"],
```
Comment (mertkahyaoglu) [inline package.json]: Let's make it `warning`.

File: src/components/Breadcrumbs.js (hunk 1 above)
Comment (ashmaroli) [inline src/components/Breadcrumbs.js]: fixed

**Pull request merged**